ruby/inc/leatherman/ruby/api.hpp

```diff
@@ -209,11 +213,11 @@ namespace leatherman {  namespace ruby {
         /**
          * See MRI documentation.
          */
-        VALUE (* const rb_num2ulong)(VALUE);
+        LONG_LONG (* const rb_num2ull)(VALUE);
```

Comment:
Should this remain `VALUE` as it is supposed to be unsigned (or have a `ULONG_LONG`)?